conanfile.py

```diff
@@ -43,6 +43,10 @@ class CuraEngineConan(ConanFile):
         "with_cura_resources": False,
     }
 
+    @property
+    def _min_cppstd(self):
+        return 20
```

Comment:
There is already a similar check in `validate()`, moreover this doesn't seem to be valid conan code 🤔